palm/plugins/dbt/dbt_containerizer.py

```diff
@@ -0,0 +1,92 @@
+from pathlib import Path
+from typing import Optional
+from palm.containerizer import PythonContainerizer
+import sys
+from palm.palm_exceptions import AbortPalm
+import click
+
+class DbtContainerizer(PythonContainerizer):
+    """
+    Containerizer for dbt projects.
+    """
+    def __init__(self, ctx, template_dir: Path, dbt_version: Optional[str] = '0.21.0') -> None:
+        self.ctx = ctx
+        self.project_name = ctx.obj.palm.image_name
+        self.template_dir = template_dir
+        self.dbt_version = dbt_version
+        self.package_manager = ''
+        self.profiles_file = ''
+    
+    def run(self) -> None:
+        """Runs the containerizer.
+
+        Returns:
+            None
+        """
+        super().check_setup()
+        self.package_manager = super().detect_package_manager()
+        self.profiles_file = self.detect_profiles_file()
```

Comment:
I don't think assignment is necessary here?

Reply:
I wasn't sure how I was going to implement this or if I needed to check for yaml v yml. I'll delete it for now and we can add it back later if it becomes an issue.

Reply:
I just mean that the assignment to `self.profiles_file` seems unnecessary. Should be enough to just call `self.detect_profiles_file()`, AFAIK we don't need to do anything with the return value of that method?

Reply:
Yes, I wasn't sure if we would want to come back later and check to see if it was valid but a) handled that to a certain extent with the templating and b) that seems outside the scope of this PR.